q2_annotate/kaiju/database.py

```diff
@@ -98,6 +103,109 @@ def _find_latest_db_url(response: bytes, database_type: str) -> str:
     raise ValueError(f"URL for database type '{database_type}' not found.")
 
 
+def _build_kaiju_bwt_index(fasta_file: str, output_dir: str, threads: int):
+    """
+    Build BWT index using kaiju-mkbwt.
+    
+    Args:
+        fasta_file: Path to input FASTA file
+        output_dir: Directory where index files will be created
+        threads: Number of threads to use
+    """
+    bwt_file = os.path.join(output_dir, "kaiju_db_idx.bwt")
```

Comment:
Perhaps this should be returned by this function so that we can use it as input to the next.